src/distributions/utils.rs

```diff
@@ -308,13 +308,13 @@ macro_rules! scalar_float_impl {
 
             #[inline]
             fn to_bits(self) -> Self::Bits {
-                unsafe { ::core::mem::transmute(self) }
+                self.to_bits()
```

Comment:
This implementation is exclusively for `no_std` — I don't believe `to_bits` is available there (which is also the only reason it's defined here).

Reply:
It looks like it is [implemented in `core`](https://doc.rust-lang.org/nightly/src/core/num/f64.rs.html#429-431) and the `no_std` tests seem to work.

Reply:
Great; strange that [the API only shows `std` versions](https://doc.rust-lang.org/core/index.html?search=to_bits). Can we remove this method completely?

Reply:
> Great; strange that the API only shows std versions.

I think this is a rustdoc "feature", it seem to apply to all `core` APIs reexported in `std`.

> Can we remove this method completely?

Yes, I just did that.